### ISSUE TITLE
[LTA] !%hash<a>:exists doesn't suggest to use %hash<a>:!exists 

### ISSUE THREAD

**Issue by titsuki** [rakudo/rakudo#2098]
## The Problem
`!%hash<a>:exists` doesn't suggest to use `%hash<a>:!exists`

## Expected Behavior
The below error message could suggest to use `%hash<a>:!exists` instead of `!%hash<a>:exists`.

## Actual Behavior
```
22:47 (titsuki) m: my %dict; say !%dict<a>:exists;
22:47 (camelia) rakudo-moar e9e8cd550: OUTPUT: «Unexpected named argument 'exists' passed␤  in block <unit> at <tmp> line 1␤␤»
```

## Steps to Reproduce

I already mentioned.

## Environment
camelia@2018/06/13 01:15

**Comment by zoffixznet** [comment]
Related: [`R#1418` Special-case the adverbs placed directly after a postfix/postcircumfix](https://github.com/rakudo/rakudo/issues/1418)

**Comment by lizmat** [comment]
Fixed this particular case with 9a45b2b8f9 .  I'm not sure how we can test this without getting too specific, since it was dying already.

I suggest we close this ticket and leave R#1418 open.
